openpype/settings/entities/schemas/projects_schema/schema_project_kitsu.json

```diff
@@ -46,16 +46,40 @@
                         {
                             "type": "boolean",
                             "key": "set_status_note",
-                            "label": "Set status on note"
+                            "label": "Set status with note"
                         },
                         {
                             "type": "text",
                             "key": "note_status_shortname",
-                            "label": "Note shortname"
+                            "label": "Status shortname"
+                        },
+                        {
+                            "type": "dict",
+                            "collapsible": true,
+                            "checkbox_key": "enabled",
+                            "key": "CustomCommentTemplate",
+                            "label": "Custom Comment Template",
+                            "children": [
+                                {
+                                    "type": "boolean",
+                                    "key": "enabled",
+                                    "label": "Enabled"
+                                },
+                                {
+                                    "type": "label",
+                                    "label": "Kitsu supports markdown and here you can create a custom comment template.<br/>You can use data from your instance's anatomy."
```

Comment:
Technically it's not using data from instance's anatomy but the publishing instance data. The anatomy data would be `instance.data["anatomyData"]` or something like that, right?

Reply:
Yeah that is more technically correct, the best type of correct :)